link_bot_gazebo/LinkBotAction action
---
link_bot_gazebo/Objects objects
bool needs_reset